scripts/main.js

```diff
@@ -0,0 +1,137 @@
+// 1.1
+var nums = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+function getEvenNums1(arr) {
+  if (!Array.isArray(arr)) throw new TypeError;
+
+  let evNums1 = [];
+  for (let i = arr.length - 1; i >= 0; i--) {
+    // Skip if element is not a number.
+    if (typeof arr[i] !== 'number') {
+      continue;
+    }
+
+    if (arr[i] % 2 != 0) continue;
+    evNums1[evNums1.length] = arr[i];
```

Comment:
it's better to just use evNums1.push(arr[i])

Reply:
I use it just because first part of each task should be done using loop and without default array methods